rcgen/Cargo.toml

```diff
@@ -38,9 +38,9 @@ zeroize = { version = "1.2", optional = true }
 [features]
 default = ["crypto", "pem", "ring"]
 crypto = []
-aws_lc_rs = ["crypto", "dep:aws-lc-rs"]
+aws_lc_rs = ["crypto", "dep:aws-lc-rs", "aws-lc-rs/aws-lc-sys"]
 ring = ["crypto", "dep:ring"]
-fips = ["aws_lc_rs", "aws-lc-rs?/fips"]
+fips = ["crypto", "dep:aws-lc-rs", "aws-lc-rs/fips"]
```

Comment:
So I solved it a bit differently, otherwise `fips` would pull in `aws-lc-sys` as well, let me know if this is fine.

Reply:
Makes sense, I forgot there was FIPS support wired up here :+1: